ngraph/CMakeLists.txt

```diff
@@ -306,20 +313,25 @@ if (NGRAPH_ONNX_IMPORT_ENABLE)
 
     set(BEFORE_ONNX_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
     set(BUILD_SHARED_LIBS ON)
+    set(BUILD_STANDALONE_STATIC OFF)
```

Comment:
Now there are 2 versions of Protobuf: dynamic (for onnx import/export) and static (for PDPD, TF and others). Actually having one shared protobuf leads to various errors when loading/unloading plugins multiple times. Example is discussed here: https://stackoverflow.com/questions/37051635/several-shared-object-using-same-proto-leading-the-the-error-file-already-exist
And this error happens for our case when we create/destroy FrontEndManager.
Ideally it would be good to have only 'static' version of protobuf, however currently onnx_importer & onnx_editor have issues with that, they spend a lot of time to find necessary workarounds and came to 'dynamic' protobuf version. I decided to not spend a lot of time again to move 'onnx' part to static protobuf and create 'static' version for PDPD (and future TF) frontends.
I believe ONNX team will have issues with future 'onnx' frontend plugin, and they'll probably have to come back to 'static' version. On that time, we'll remove building of dynamic protobuf and keep only 'static' version

Reply:
Going back to static protobuf required by the onnx_importer will most probably bring back the problems caused by static version of protobuf. This will be a big regression.

Please test this PR against the ONNX importer unit tests and integration with ONNXRuntime(including windows) and make sure that the tests still pass and there are no segfaults.

@postrational FYI

Reply:
ONNX currently still uses 'dynamic protobuf', no changes are there. Only PDPD frontend uses static protobuf version and it doesn't conflict with any other components which uses dynamic protobuf. If we enable static protobuf for ONNX - we'll clearly get CI failures now, but this PR is green

Reply:
>I believe ONNX team will have issues with future 'onnx' frontend plugin, and they'll probably have to come back to 'static' version. On that time, we'll remove building of dynamic protobuf and keep only 'static' version
1. As I know @slyalin do some experiments with onnx_importer/onnx_editor based on dynamic protobuf and it works correctly. Am I right?
2. As I understand in the current stack: we have dynamic protobuf used by onnx_importer and static by PDPD at the same time. Could you create a draft with `NGRAPH_PDPD_FRONTEND_ENABLE=ON` on all CI where `NGRAPH_ONNX_IMPORT_ENABLE=ON`? (probably it is worth to consider if some of them should have `NGRAPH_PDPD_FRONTEND_ENABLE=ON` permanently)

Reply:
> Could you create a draft with NGRAPH_PDPD_FRONTEND_ENABLE=ON on all CI where NGRAPH_ONNX_IMPORT_ENABLE=ON?
@mbencer Yes, NGRAPH_PDPD_FRONTEND_ENABLE is enabled in this PR everywhere where NGRAPH_ONNX_IMPORT_ENABLE=ON. See line 64. Also you can verify that PDPD frontend is enabled in CI 'cmake' logs (I added 'message' prints on line 116 in this file)

Reply:
@nosovmik that's right, but some configurations are defined in https://github.com/openvinotoolkit/openvino/tree/master/.ci and some by external code (we can discuss offline). I've pushed for test: https://github.com/openvinotoolkit/openvino/pull/6274

Reply:
@mbencer @tomdol Do you have any objections against the current solution? Can we get conflicts?

Reply:
@mbencer @tomdol Guys, could you please comment?

Reply:
Sorry, I missed the GH notification. I still think that going back to the static PB in the future will bring the previous problems back. According to this answer https://stackoverflow.com/a/37056328/4517806 it's possible to use static protobuf but each library will have to be linked with a copy of PB. This will increase the size of each library and will require some extra serialization/deserialization pairs. For example The model will have to be deserialized in the ONNX editor, modified, serialized to a string so that it can be safely passed to the importer, deserialized in the latter and only then converted to an ngraph function. I'm still not 100% sure that will work.

I definitely don't want to block this PR and my approval is not required here. I suggest consulting it with @postrational because he's in charge of the FE API integration for the onnx_importer.

Reply:
Ok, I propose to return back to this discussion if we have any issues with ONNX Frontend.

Reply:
Based on CI results (#6274) having two instances of protobuf (shared for onnx and static for pdpd) doesn't cause problems on any tested platform.
I also downloaded this branch and prepare unit tests where both pdpd and onnx_importer are used in the same process. At least on my dev machine, I haven't observed problems (we should have such tests in CI after ONNXImporter/FrontendApi integration).
The code with double protobuf is not very elegant, but I don't know how to deal with it now.

Reply:
Based on CI results (#6274) having two instances of protobuf (shared for onnx and static for pdpd) doesn't cause problems on any tested platform.
I also downloaded this branch and prepare unit tests where both pdpd and onnx_importer are used in the same process. At least on my dev machine, I haven't observed problems (we should have such tests in CI after ONNXImporter/FrontendApi integration).
The code with double protobuf is not very elegant, but I don't know how to deal with it now.